From 481c089b381b4a4138a397ebc646a4cafb3d2400 Mon Sep 17 00:00:00 2001
From: John Zhuge <jzhuge@cloudera.com>
Date: Thu, 6 Jul 2017 23:24:55 -0700
Subject: [PATCH 2533/2748] HDFS-11861. ipc.Client.Connection#sendRpcRequest
 should log request name. Contributed by John
 Zhuge.

(cherry picked from commit 5672ae7b37ce75086a1cb5bb9a388288fc913eb7)

Conflicts:
	hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java

Change-Id: I0101bcc8f41e7a37ef49fb35ccf5a20daa9aa96a
---
 .../main/java/org/apache/hadoop/ipc/Client.java    |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java
index 2b17923..5752848 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java
@@ -1060,8 +1060,10 @@ public void run() {
                   return;
                 }
                 
-                if (LOG.isDebugEnabled())
-                  LOG.debug(getName() + " sending #" + call.id);
+                if (LOG.isDebugEnabled()) {
+                  LOG.debug(getName() + " sending #" + call.id
+                      + " " + call.rpcRequest);
+                }
          
                 byte[] data = d.getData();
                 int totalLength = d.getLength();
-- 
1.7.9.5

